src/Symfony/Bundle/SecurityBundle/Command/InitAclCommand.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\SecurityBundle\Command;
 
-@trigger_error(sprintf('Class "%s" is deprecated since version 3.4 and will be removed in 4.0. Use Symfony\Bundle\AclBundle\Command\SetAclCommand instead.', SetAclCommand::class), E_USER_DEPRECATED);
+@trigger_error(sprintf('Class "%s" is deprecated since version 3.4 and will be removed in 4.0. Use Symfony\Bundle\AclBundle\Command\SetAclCommand instead.', InitAclCommand::class), E_USER_DEPRECATED);
```

Comment:
I'd move this below the `use` statements and use `::class` for both classes.

Reply:
Wait my fix was incomplete it seems.